ghost/admin/app/templates/settings/general.hbs

```diff
@@ -18,7 +18,7 @@
 
     <div>
         <div class="gh-main-section">
-            <h4 class="gh-main-section-header small bn">Publication info</h4>
+            <h4 class="gh-main-section-header small bn">{t('Publication info')}</h4>
```

Comment:
```suggestion
            <h4 class="gh-main-section-header small bn">Publication info</h4>
```

This is not needed 🙂

Reply:
I'm trying to figure out the scope of what is being translated. The "Publication info" heading is clearly something in English that is being shown to users. Is there an easy way to distinguish why something like this isn't supposed to be translated?

Reply:
This is the Admin client, which is out of scope as we're only translating Portal and member emails right now

Reply:
Hello, I have fixed it and have committed the changes.